### DISCUSSION TITLE
Benefits not available yet

### DISCUSSION THREAD

**Discussion by MaxiC-UADE** [community/community#137070] [GitHub Education]
### Hi everyone,

I have been approved for GitHub Education on Aug, 21 but my benefits are not available yet.

**Comment by davevad93** [comment]
Hi @MaxiC-UADE ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @MaxiC-UADE was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by MaxiC-UADE** [comment]
Thank you!

**Reply by davevad93** [reply]
You're welcome @MaxiC-UADE ! 👍

**Comment by nikeetan** [comment]
Hello Everyone,
Benefits are unavailable for me as well could you please help me to resolve the same I have been approved on 26th Aug
